golang/cosmos/x/vstorage/client/cli/query.go

```diff
@@ -38,9 +39,20 @@ func GetCmdGetData(queryRoute string) *cobra.Command {
 
 			path := args[0]
 
-			res, err := queryClient.Data(cmd.Context(), &types.QueryDataRequest{
-				Path: path,
-			})
+			var res proto.Message
+
+			// if path ends with '/' then remove it
+			if path[len(path)-1] == '/' {
```

Comment:
`'/'` seems sort of arbitrary for a vstorage path.  I'd expect something more coherent to that model, like a trailing `'.'`.  WDYT?